cone_detections_topic_name: /perception/lidar/cone_detections

lidar_topicname: /carla/ego_vehicle/radar_front

node_rate: 10   # [Herz]